content/applications/sales/sales/amazon_connector/features.rst

```diff
@@ -1,28 +1,29 @@
 =========================
-Amazon Connector Features
+Amazon Connector features
 =========================
 
-The **Amazon Connector** synchronizes the orders between Amazon and your Odoo database, which
-reduces considerably the amount of time spent on your Amazon Seller Central dashboard, making your
-daily routine a lot easier.
+The *Amazon Connector* synchronizes orders between Amazon and Odoo, which considerably reduces the
+amount of time spent on the Amazon Seller Central dashboard.
 
-Supported Features
+Supported features
 ==================
 
-The connector is able to:
+The *Amazon Connector* is able to:
 
-- Synchronize (Amazon to Odoo) all confirmed orders (both FBA and FBM) and their order items which
+- Synchronize (Amazon to Odoo) all confirmed orders (both FBA and FBM), and their order items, which
   include:
 
-  - the product’s name, description and quantity
-  - the shipping costs for the product
-  - the gift wrapping charges
+  - product name, description, and quantity
+  - shipping costs for the product
+  - gift wrapping charges
 
-- Create on Odoo any missing partner related to an order (contact types supported: contact and
+- Create any missing partner related to an order in Odoo (contact types supported: contact and
   delivery).
-- Notify Amazon of a shipping confirmed on Odoo (FBM) in order to get paid.
+
+- Notify Amazon of confirmed shipping in Odoo (FBM) to get paid.
 
 - Support multiple seller accounts.
+
 - Support multiple marketplaces per seller account.
 
 +----------------------+----------------------------+-------------------------------------+
```

Comment:
This table needs a lot of clarification. I don't understand what the "Shipping" row is saying---shipping charges and delivery are handled by Odoo? What does "Handled by the delivery" mean in the "Stock Management" row? What does "Notify Amazon when confirming delivery" mean---that Odoo does this automatically, or you need to do it manually?

This table could use more descriptive entries and/or some kind of introductory sentence that tells you how to read it, like "The following table lists capabilities provided by Odoo when using the Amazon Connector," or something like that.

Reply:
My apologies. Again, this is one of those elements from the previous doc that I felt inclined to replicate, despite not being able to fully ingratiate myself in a paid-for Amazon Seller account. I assumed, since the previous version had been reviewed/approved, that this would be permissible. But I fully understand (and agree) that it needs to be clarified. I also agree with your suggestion to get some detailed explanation about this from @dalaOdoo and/or @Demesmaeker ...

So, if either of you could kindly provide me with some clarification on this included table -- and double-check it for accuracy purposes -- that would be greatly appreciated. Once we get that, we can complete this doc, and push it to the next stage. Thanks for all the continued help with these Amazon docs! 👍

Reply:
Hello!
Orders: which orders are synced based on their status. For instance, shipped orders in FBM aren't synchronized.  Canceled orders though are cancelled if they are existing in the DB, they aren't sync if they weren't created.

Shipping: The charges, as sent by Amazon, are applied on the sale orders. For FBM, a Delivery order is automatically created, so we can continue the flow.

Gift Wrapping: for FBM, it's also shown on the sale orders. The charges (the price of the gift wrapping) are added to the SO, as well as the message if any, so that the package could have every needed information without the user having to go through Amazon backend

Stock management: the stock is updated when the products are sold, in case of FBA, stock in the Amazon virtual location decreased, in the FBM, the stock is managed with the delivery order, and can be synchronized directly from Odoo too. Meaning that if you sold some item in your shop, the number of available items will be sent to Amazon, so that the number of available item in Amazon will also decreased. (Opposite is also true, when you refurnish your stock)

Confirmation: when you confirm a delivery, the information about the carrier and the tracking reference are automatically send to Amazon, so that the order is mark as sent by Amazon with all needed information without going through Amazon backend.

Reply:
Nice feature that isn't written down (and isn't even explain in "Synchronization of Orders") is that we also now have the possibility to sync a single order by it's reference, both in FBM and FBA. So this note is a bit irrelevant now: 
![image](https://github.com/odoo/documentation/assets/69820391/3d4e2a9e-b206-4077-99fe-d5eb6ab0a4a3)

Reply:
@ksc-odoo & @meng-odoo here is a suggestion of change for this table 
https://docs.google.com/spreadsheets/d/1kIdkYIKYOLT2aFed4QBPiaRFg6f79KF0iIz6c_C2Au8/edit?usp=sharing

Reply:
@Demesmaeker Thanks for the helpful spreadsheet! I really appreciate it. I'll also go back to the PR re: "Manage Amazon Orders in Odoo..." to remove that note and add a mention about how it's now possible to synch a single order by its reference in FBM and FBA... 👍